README.md

```diff
@@ -1,45 +1,53 @@
 
-![Atala Prism Logo](Logo.png)
+The following will explain how to use the SDK in your project, how to prepare your development environment if you wish
+to contribute and some basic considerations around the project.
 
-# Welcome to Atala PRISM Kotlin multiplatform SDK
-
-The following will explain how to use the SDK in your project, how to prepare your development environment if you wish to contribute and some basic considerations around the project.
-
-This SDK provides a library and documentation for developers to build Android and JVM connected SSI applications with Atala PRISM.
+This SDK provides a library and documentation for developers to build Android and JVM connected SSI applications with
+Identus.
 
 ## Basic considerations
 
-### Atala PRISM
+### Identus
 
-Atala PRISM is a self-sovereign identity (SSI) platform and service suite for verifiable data and digital identity. Built on Cardano, it offers core infrastructure for issuing DIDs (Decentralized identifiers) and verifiable credentials, alongside tools and frameworks to help expand your ecosystem.
+Identus is a self-sovereign identity (SSI) platform and service suite for verifiable data and digital identity. Built on
+Cardano, it offers core infrastructure for issuing DIDs (Decentralized identifiers) and verifiable credentials,
+alongside tools and frameworks to help expand your ecosystem.
 The complete platform is separated in multiple repositories:
 
-* [atala-prism-wallet-sdk-swift](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift) - Repo that implements Atala PRISM for Apple platforms in Swift.
-* [atala-prism-wallet-sdk-ts](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) - Repo that implements Atala PRISM for Browser and Node.js platforms in Typescript.
-* [atala-prism-building-blocks](https://github.com/hyperledger-labs/open-enterprise-agent) - Repo that contains the platform Building Blocks.
+* [atala-prism-wallet-sdk-swift](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift) - Repo that implements
+  Identus for Apple platforms in Swift.
+* [atala-prism-wallet-sdk-ts](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) - Repo that implements
+  Identus for Browser and Node.js platforms in Typescript.
+* [atala-prism-building-blocks](https://github.com/hyperledger-labs/open-enterprise-agent) - Repo that contains the
+  platform Building Blocks.
 * [atala-prism-mediator](https://github.com/input-output-hk/atala-prism-mediator) - Repo for DIDComm V2 Mediator
 
 ### Modules / APIs
 
-Atala PRISM Kotlin multiplatform SDK provides the following building blocks to create, manage and resolve decentralized identifiers, issue, manage and verify verifiable credentials, establish and manage trusted, peer-to-peer connections and interactions between DIDs, and store, manage, and recover verifiable data linked to DIDs.
+Identus Kotlin multiplatform SDK provides the following building blocks to create, manage and resolve decentralized
```

Comment:
`Identus KMM SDK` should be `Edge Agent SDK KMM` according to the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0)